TYPO3.Neos/Resources/Private/Templates/Media/Asset/RelatedDocumentNodes.html

```diff
@@ -0,0 +1,46 @@
+{namespace neos=TYPO3\Neos\ViewHelpers}
+<f:layout name="EditImage"/>
+
+<f:section name="Title">RelatedDocumentNodes view of AssetController</f:section>
+
+<f:section name="Content">
+      <legend>{asset.resource.filename}</legend>
+
+      <table class="neos-info-table">
+        <thead>
+          <tr>
+            <th>{neos:backend.translate(id: 'media.relatedDocumentNodes.document', source: 'Modules', package: 'TYPO3.Neos')}</th>
+            <th>{neos:backend.translate(id: 'media.relatedDocumentNodes.reference', source: 'Modules', package: 'TYPO3.Neos')}</th>
+            <th class="neos-aRight">{neos:backend.translate(id: 'media.field.lastModified', source: 'Modules', package: 'TYPO3.Neos')}</th>
+          </tr>
+        </thead>
+        <tbody>
+          <f:for each="{relatedDocumentNodes}" as="relatedDocumentNode">
+            <tr>
+              <td><neos:link.node node="{relatedDocumentNode.documentNode}" /></td>
```

Comment:
here it would be too to show the node type icon with the node type name as the title (hover) – you can copy this https://github.com/neos/neos-development-collection/pull/420/files